assets/js/mbta.js

```diff
@@ -61,769 +61,4535 @@ const branchConfig = {
 };
 
 const stationConfig = {
-  Orange: [
-    {
-      id: 'OOAK',
-      name: 'Oak Grove',
-      zones: {
-        n: false, s: 'Platform', e: 'Busway', w: false, c: false, m: true,
+  Orange: {
+    batchModes: { auto: true, headway: true, off: true },
+    stations: [
+      {
+        id: 'OOAK',
+        name: 'Oak Grove',
+        zones: {
+          n: {
+            value: false,
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
```

Comment:
I assume there's no good way to avoid writing this out a bunch of times over while still obeying the PropTypes set elsewhere?

Reply:
No, I suppose I could set a default in the component. I figured explicitly listing all the keys is similar to how we handle the `zones`, as opposed to only including the `zones` with values.

Reply:
Hmm... I suppose with `zones` there's less repetition whereas with `modes` they're pretty much all the same, except for the busway signs. I don't have a strong preference though.